lib/Remotery.c

```diff
@@ -8892,6 +8960,26 @@ RMT_API rmtSampleType _rmt_SampleGetType(rmtSample* sample)
     return sample->type;
 }
 
+RMT_API rmtStatType _rmt_SampleGetStatType(rmtSample* sample)
+{
+    return sample->stat_info.type;
+}
+
+RMT_API const char* _rmt_SampleGetStatDesc(rmtSample* sample)
+{
+    return StringTable_Find(g_Remotery->string_table, sample->stat_info.desc);
+}
+
+// TODO: How to deal with getting wrong type?
+// SHould it return an rmtError instead?
+RMT_API rmtI32 _rmt_SampleGetStatValueI32(rmtSample* sample)
+{
+    if (sample->stat_info.type != RMT_StatType_I32)
+        return 0;
+    return sample->stat_info.value.ivalue;
+}
```

Comment:
Q: Should we use `rmtError` instead, or is it ok with returning 0.
I think keeping it simple with returning 0 should suffice.

Reply:
0 sounds good.